src/dataflow-types/src/client.proto

```diff
@@ -22,20 +22,22 @@ import "google/protobuf/empty.proto";
 
 package mz_dataflow_types.client;
 
+message ProtoInstanceConfig {
+    int64 replica_id = 1;
+    mz_dataflow_types.logging.ProtoLoggingConfig logging = 2;
```

Comment:
Let's switch the two around (i.e., make `logging` member 1) for backwards-compatibility -- I know it's not serialized, but it's a good habit.

Reply:
So it still is backwards compatible even if we rename and move message types, as long as their structure remains the same?